tools/run_tests/performance/scenario_config.py

```diff
@@ -830,10 +832,23 @@ def scenarios(self):
         rpc_type='STREAMING', client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
         server_language='c++', async_server_threads=1)
 
+    # TODO(ddyihai): Investigate why when async_server_threads=1/CPU usage 340%, the QPS performs
+    # better than async_server_threads=0/CPU usage 490%.
+    yield _ping_pong_scenario(
+        '%s_to_cpp_protobuf_sync_unary_qps_unconstrained' % php_extension_mode,
+        rpc_type='UNARY', client_type='SYNC_CLIENT', server_type='ASYNC_SERVER',
+        server_language='c++', outstanding=1, async_server_threads=1, unconstrained_client='sync')
+
+    yield _ping_pong_scenario(
+        '%s_to_cpp_protobuf_sync_streaming_qps_unconstrained' % php_extension_mode,
+        rpc_type='STREAMING', client_type='SYNC_CLIENT', server_type='ASYNC_SERVER',
+        server_language='c++', outstanding=1, async_server_threads=1, unconstrained_client='sync')
+
   def __str__(self):
+    if self.use_protobuf_c_extension:
```

Comment:
this can become either just "php"  or    selection from {"php","php_protobuf_c"} depending on what you end up needing.

Reply:
Change to  "self.php_protobuf_c"